charts/tfy-workflow-propeller/values.yaml

```diff
@@ -0,0 +1,72 @@
+## @section Configuration for tfy-workflow-propeller
+##
+
+## @section flyte-core configurations
+flyte-core:
+  common:
+    ingress:
+      ## @param flyte-core.common.ingress.enabled to enable the ingress
+      enabled: false
+  secrets:
+    adminOauthClientCredentials:
+      ## @param flyte-core.secrets.adminOauthClientCredentials.enabled to enable Oauth client credentials
+      enabled: true
+  storage:
+    ## @param flyte-core.storage.type to define the storage type
+    type: <to_be_provided>
+    limits:
+      ## @param flyte-core.storage.limits.maxDownloadMBs to define the max download size
+      maxDownloadMBs: 10
+    ## @param flyte-core.storage.bucketName to define the storage bucket name
+    bucketName: <to_be_provided>
+    connection:
+      ## @param flyte-core.storage.connection.region to define the storage connection region
+      region: <to_be_provided>
+      ## @param flyte-core.storage.connection.auth-type to define the storage connection auth type
+      auth-type: <to_be_provided>
+    ## @param flyte-core.storage.enable-multicontainer to enable the multicontainer
+    enable-multicontainer: true
+  webhook:
+  ## @param flyte-core.webhook.enabled to enable the webhook
+    enabled: true
+  configmap:
+    core:
+      propeller:
+        ## @param flyte-core.configmap.core.propeller.metadata-prefix to set the storage uri path to set store the metadata, its values should be bucketName/metadata
+        metadata-prefix: <to_be_provided>\workflows\metadata
+        ## @param  flyte-core.configmap.core.propeller.rawoutput-prefix to set the storage uri path to set store the raw output, its values should be bucketName/raw_data
+        rawoutput-prefix: <to_be_provided>\workflows\raw_data
+        ## @param flyte-core.configmap.core.propeller.publish-k8s-events to publish kube events
```

Comment:
where is admin section of configmap?

How will the propeller even connect to it?

Please strictly stick to these values: https://app.devtest.truefoundry.tech/deployments/clywsv9xw00g301t2fmwadrcc?tab=manifest

We want exactly this and nothing else!!